sacct, scancel, srun: add page (#4600)
tldr-pages/tldr: pages/linux/sacct.md

# sacct

> Display accounting data from the Slurm service.
> More information: <https://slurm.schedmd.com/sacct.html>.

> 

Reviewer: Could you add the example for just `sacct` without params and `sacct --brief` (looks useful to see what is running)?

> - Check basic items such as elapsed time, job name, cpus requested, and memory requested:

Reviewer: ```suggestion
- Display elapsed time, job name, number of requested CPUs, and memory requested of a job:
```

> `sacct -j {{job_id}} --format=elapsed,jobname,reqcpus,reqmem`

Reviewer: We prefer the long argument format and the argument options should be between `{{}}`:
```suggestion
`sacct --jobs={{job_id}} --format={{elapsed}},{{jobname}},{{reqcpus}},{{reqmem}}`
```

Author: With `sacct` the `--format` option allows you to specify different fields you want to be displayed, in this case, elapsed, jobname, requested CPUs, and requested memory. I am double checking that the curly brackets around the fields would be correct in this situation, or would it be better to use:
`sacct --jobs={{job_id}} --format={{elapsed,jobname,reqcpus,reqmem}}`

 I have resolved the other comments in the latest commit.

Reviewer: Checking the `tokei` page, is `{{}}` for each field.
